coordinator/src/node.rs

```diff
@@ -66,25 +63,31 @@ impl Node {
     async fn open_position(&self, trade_params: &TradeParams) -> Result<()> {
         tracing::info!("Opening position");
 
-        // todo: Revisit position model and store to database.
-        let position = Position {
-            contract_symbol: trade_params.contract_symbol,
+        let margin_trader = margin_trader(trade_params);
+        let margin_coordinator = margin_coordinator(trade_params);
+
+        let tomorrow = OffsetDateTime::now_utc().date() + Duration::days(1);
+        let expiry = tomorrow.midnight().assume_utc();
```

Comment:
Just to recap that I got this correctly: 

This means if I open the position 

at `Mo 23:59` it will be closed at `Tue midnight`
at `Mo 00:01` it will be closed at `Tue midnight`

so it will be open for at least 24h, up to 47 hours and 59 mins.

Reply:
Note: It should *not* be between 1sec and 24h! - the user should always have *at least* 24h, otherwise the trading experience is not useful.

Example: If I trade at Mon 09:59am in Sydney I would hope my positions stays open until Tue 10am and not until Mon 10am ;)

Reply:
It would be great to (eventually) have a test for this time logic. Feel free to merge and add once we go for actually closing...

Inspiration:

https://github.com/get10101/itchysats/blob/master/crates/model/src/cfd.rs#L2904-L2997